nextflow.config

```diff
@@ -1,8 +1,73 @@
-singularity.enabled = false
-singularity.autoMounts = false
+profiles {
 
-docker.enabled = true
-process.container = 'haniffalab/webatlas-pipeline:0.3.2'
+  conda {
+    executor = 'local'
+    singularity.enabled = false
+    docker.enabled = false
+    conda.enabled = true
+    process.conda = "${projectDir}/docker/conda.yaml"
+    conda.useMamba = false
+  }
+
+  docker {
+    executor = 'local'
+    singularity.enabled = false
+    singularity.autoMounts = false
+
+    docker.enabled = true
+    process.container = 'haniffalab/webatlas-pipeline:0.3.2'
+  }
+
+  sanger_lsf {
+    process.container = 'haniffalab/webatlas-pipeline:0.3.2'
+
+    docker.enabled = false
+    singularity {
+        enabled = true
+        autoMounts = false
+        runOptions = '--bind /lustre --bind /nfs --cleanenv --home /tmp'
+    }
+    executor {
+        name = 'lsf'
+        perJobMemLimit = true
+        container = '/lustre/scratch127/cellgen/cellgeni/tickets/tic-2248/actions/docker--haniffalab--vitessce-pipeline--0.0.1.sif'
+        jobName = { "$task.hash" }
+        queue = 'imaging'
+    }
+  }
+}
+
+process {
+
+    withName: image_to_zarr {
+        memory = 20.GB
+        cpus = 4
+    }
+
+    withName: ome_zarr_metadata {
+        memory = 2.GB
+        cpus = 1
+    }
+
+    withName: route_file {
+        memory = 4.GB
+        cpus = 1
+    }
+
+    withName: Build_config {
```

Comment:
At some point you and @dannda should have a chat about normalizing the processes/workflow names (i vote for snake_case but I guess it makes mroe sense for them to be cammelCase, perhaps?)

Reply:
Yea, I prefer snake_case as well. I will amend that in the different PR